bool is_gaze
int32 face_ctr_x
int32 face_ctr_y
int32 face_pan
int32 face_tilt
int32 face_roll
int32 smile
